tests/example_tests/test_libuserd_exhaustive.py

```diff
@@ -564,6 +784,20 @@ def compare(userd, session, file1_userd, file2_userd, file1_session, file2_sessi
     # Test variable lists
     variable_failures, variable_error_code = _test_variables(variables_userd, variables_pyensight)
 
+    # Test XYZ
+    xyz_failures, xyz_error_code = _test_xyz_values(
+        parts_pyensight, variables_pyensight, part_xyz_cache_userd, time_step
+    )
+
+    if (
+        part_element_cache_userd and part_element_cache_userd != {}
```

Comment:
in Python when you do:
if (part_element_cache_userd):

It checks both that is not None and is not empty. So the second check is redundant

Reply:
Done